packages/next/build/check-next-script-import.ts

```diff
@@ -0,0 +1,16 @@
+import { promises } from 'fs'
+import { join } from 'path'
+
+export async function checkNextScriptImport(
+  pagePaths: string[],
+  pagesDir: string
+): Promise<boolean> {
+  for (const page of pagePaths) {
+    const fileContent = await promises.readFile(join(pagesDir, page), 'utf8')
+    if (/import[^}]*.*('|")next\/script('|")/.test(fileContent)) {
```

Comment:
The pattern here can not match the following cases:

```
import Script from 'next/dist/client/script';
const Script = require('next/script');
```

IMHO we could simply use `!fileContent.includes('next/script') && !fileContent.includes('next/dist/client/script')`.